pkgs/by-name/a-/a-keys-path/package.nix

```diff
@@ -0,0 +1,116 @@
+# copied from oh-my-git/package.nix by @jojosch (Johannes Schleifenbaum)
+# added in nixos/nixpkgs#119642
+{
+  lib,
+  autoPatchelfHook,
+  copyDesktopItems,
+  fetchFromGitHub,
+  makeDesktopItem,
+  stdenv,
+  alsa-lib,
+  godot3-export-templates,
+  godot3-headless,
+  libX11,
+  libXcursor,
+  libXext,
+  libXi,
+  libXinerama,
+  libXrandr,
+  libXrender,
+  libglvnd,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "a-keys-path";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "geegaz";
+    repo = "A-Key-s-Path";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-i9INPB883/0fmtZgJpVpsA/7Q51DHOMWvkLF9hKE6gQ=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    copyDesktopItems
+    godot3-headless
+  ];
+
+  buildInputs = [
+    libX11
+    libXcursor
+    libXext
+    libXi
+    libXinerama
+    libXrandr
+    libXrender
+    libglvnd
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "a-keys-path";
+      exec = "a-keys-path";
```

Comment:
Shouldn't this use the full path?

Reply:
not many applications are doing it, but I added it.

Reply:
after doing a nixos-rebuild switch, does the desktop environment reload these?

Reply:
@nbraud could you re-review?

Reply:
I am changing it back to not use the absolute nix store path but just the executable name.

I've tried it in a vm with xfce desktop and whiskermenu, and the desktop had to be restarted for the desktopItem to reflect the new package after a switch.
But with the executable name, the new version will be launched when launched from the applications menu, without the need to restart the desktop.